validation-plugin.js

```diff
@@ -0,0 +1,160 @@
+module.exports = async ({ github, context, core, probe }) => {
```

Comment:
```suggestion
module.exports = async ({ github, context, core }) => {
```

Reply:
We need this here, related to: <https://github.com/obsidianmd/obsidian-releases/pull/1346/files#r1026772896>